### PR TITLE
Added Romanian translations.

### PR DESCRIPTION
**Pull request by jladage**
Just added translations for Romanian language.

### CODE REVIEW

**Comment by icemac** [comment]
Thank you for your contribution.

According to the contributing policies of the zopefoundation organization you need to sign a contributor agreement before any non-trivial change can be merged. For details please consult the [Contributing guidelines for zopefoundation projects](https://github.com/zopefoundation/.github/blob/master/CONTRIBUTING.md).

**Comment by icemac** [comment]
@MrTango Is your Romanian good enough to review these translations?

**Comment by jladage** [comment]
Hi Micheal,I signed a contributing license for Plone in 2007. Will that do? Or do you want a new one just for Zope which uses Plone’s as I understand.Kind regards,Jean-Paul LadageOn 13 Aug 2024, at 07:08, Michael Howitz ***@***.***> wrote:﻿
@MrTango Is your Romanian good enough to review these translations?

**Comment by icemac** [comment]
@jladage When you sign a Zope contributor agreement, you will get a Member of the `zopefoundation` project, this is the way I see that the agreement was signed and it will allow you to merge yout PR after successful review.

So I cannot answer your question. I'd suggest either to ask someone of the Plone Foundation or just sign a contributor agreement for Zope.

**Comment by icemac** [comment]
@jladage You are now listed as `Member` are you also able to merge yourself?

**Comment by jladage** [comment]
@icemac nope, at least not if nobody is able to review the Romanian translations I created with deepl.

**Comment by davisagli** [comment]
@avoinea Would you like to help review this or ask someone else to?

File: src/z3c/form/locales/ro/LC_MESSAGES/z3c.form.po [hunk 1]
```diff
@@ -0,0 +1,681 @@
...
+
+#: src/z3c/form/interfaces.py:558
+msgid ""
+"A collection of dictionaries containing all pieces of information for "
+"rendering. The following keys must be in each dictionary: id, value, "
+"content, selected"
+msgstr ""
+"O colecție de dicționare care conțin toate informațiile pentru redare. "
+"Următoarele chei trebuie să fie în fiecare dicționar: id, valoare, conținut, "
+"selectat"
```
Comment (avoinea) [inline src/z3c/form/locales/ro/LC_MESSAGES/z3c.form.po:317]:
```suggestion
"content, selected"
```

File: src/z3c/form/locales/ro/LC_MESSAGES/z3c.form.po [hunk 2]
```diff
@@ -0,0 +1,681 @@
...
+
+#: src/z3c/form/interfaces.py:563
+msgid "No-Value Message"
+msgstr "Mesaj fără valoare"
```
Comment (avoinea) [inline src/z3c/form/locales/ro/LC_MESSAGES/z3c.form.po:321]:
```suggestion
msgstr "Mesaj lipsă valoare"
```

File: src/z3c/form/locales/ro/LC_MESSAGES/z3c.form.po [hunk 3]
```diff
@@ -0,0 +1,681 @@
...
+"indiferent dacă sunt completate sau nu"
+
+#: src/z3c/form/interfaces.py:659
+msgid "Has required fields"
+msgstr "Are câmpuri obligatorii"
+
+#: src/z3c/form/interfaces.py:660
+msgid "A flag set when at least one field is marked as required"
+msgstr ""
+"Un steag setat atunci când cel puțin un câmp este marcat ca fiind obligatoriu"
```
Comment (avoinea) [inline src/z3c/form/locales/ro/LC_MESSAGES/z3c.form.po:380]:
```suggestion
"Setat atunci când cel puțin un câmp este marcat ca fiind obligatoriu"
```

File: src/z3c/form/locales/ro/LC_MESSAGES/z3c.form.po [hunk 4]
```diff
@@ -0,0 +1,681 @@
...
+msgid ""
+"A flag, when set, causes form actions to be updated again after their "
+"execution."
+msgstr ""
+"Un indicator, atunci când este setat, face ca acțiunile din formular să fie "
+"actualizate din nou după executarea lor."
+
+#: src/z3c/form/interfaces.py:874
+msgid "The context in which the widget is displayed."
+msgstr "Contextul în care este afișat widget-ul."
```
Comment (avoinea) [inline src/z3c/form/locales/ro/LC_MESSAGES/z3c.form.po:472]:
```suggestion
msgstr "Contextul în care este afișat widgetul."
```

**Review by avoinea — CHANGES_REQUESTED** [review]

**Comment by jladage** [comment]
Hi @avoinea, I merged your translation suggestions. Could you please review once more so I can merge?

**Review by avoinea — APPROVED** [review]

**Review by avoinea — APPROVED** [review]

**Comment by jladage** [comment]
@icemac the test coverage is currently blocking this request from merging. I'm not familiar enough with the code to understand why. Do we test translations? Are our changes the result of the test coverage problem?

**Comment by icemac** [comment]
@jladage The translations are not part of the coverage and are not tested. I saw a drop in coverage on serveral packages, probably because of a new version of the `coverage` package.

I'll merge this PR now and tackle the coverage problem later on.

**Review by icemac — APPROVED** [review]

**Pull request merged**